apps/ui/package.json

```diff
@@ -15,7 +15,9 @@
     "extends": "../../package.json"
   },
   "devDependencies": {
+    "@kampus/kampus": "*",
     "@kampus/tailwind": "*",
+    "@kampus/ui-next": "*",
```

Comment:
the `@kampus/kampus` and `@kampus/ui-next` imports exist under `apps/ui/` (see [here](https://github.com/kamp-us/monorepo/blob/d63c9eeecff6c1c55afc81872b440677994c94f7/apps/ui/stories/Avatar.stories.tsx#L3)).

these imports were working without issues with `npm` but they need to be defined as dependencies with `pnpm` as otherwise the imports will fail (due to how pnpm stores packages under `node_modules`)

Reply:
This is fine @Ketcap 

`apps/ui` is our storybook application.
`packages/ui` is the `kampus-ui` package.